### PR TITLE
add image chat demo

### PR DESCRIPTION
**Pull request by irexyc**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by lvhan028** [comment]
readme上增加一条 news 吧

File: examples/vl/app.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+        session,
+    ):
+        generator = model.create_instance()
+        history = session._message
+        sequence_start = len(history) == 1
+        seed = random.getrandbits(64) if sequence_start else None
+        input_ids, features, ranges = preprocessor.prepare_query(
+            history[-1][0], sequence_start)
+
+        if len(input_ids) + session.step > model.model.session_len:
```
Comment (AllentDan) [inline examples/vl/app.py:133]: shall we add `request output len`?

File: examples/vl/app.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    preprocessor = SUPPORTED_MODELS[args.model_name](args.hf_ckpt)
+    model = TurboMind.from_pretrained(llm_ckpt, model_name=args.model_name)
+    return preprocessor, model
+
+
+def launch_demo(args, preprocessor, model):
+
+    def add_image(chatbot, session, file):
+        chatbot = chatbot + [((file.name, ), None)]
+        # print('add_image', chatbot)
```
Comment (AllentDan) [inline examples/vl/app.py:104]: may remove comment line

File: examples/vl/app.py [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        type=str,
+        default=DEFAULT_LLM_CKPT,
+        help='LLM checkpoint name or path, default to %(default)s')
+    parser.add_argument('--server-port',
+                        type=int,
+                        default=9006,
+                        help='Server port, default %(default)s')
+    parser.add_argument('--server-name',
+                        type=str,
+                        default='127.0.0.1',
```
Comment (AllentDan) [inline examples/vl/app.py:86]: 0.0.0.0 好像比 127.0.0.1 好用，开着代理的话后者我没跑起来

**Review by AllentDan — APPROVED** [review]

File: examples/vl/xcomposer_model.py [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+meta_instruction = """meta instruction
+You are an AI assistant whose name is 浦语.
+- 浦语 is a conversational language model that is developed by Shanghai AI Laboratory (上海人工智能实验室). It is designed to be helpful, honest, and harmless.
+- 浦语 can understand and communicate fluently in the language chosen by the user such as English and 中文.
+conversation
+"""  # noqa
+
+
+@MODELS.register_module(name='internlm-xcomposer-7b')
+class InternLMXComposerTemplate(BaseModel):
```
Comment (lvhan028) [inline examples/vl/xcomposer_model.py:23]: May update docstring

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**